### PR TITLE
fix: mix up share and edit privileges on create

### PR DESCRIPTION
**Pull request by gabisavu-ionos**
## Checklist

- [ ] PR name added as appropriate (e.g. `feat:`/`fix:`/`doc:`/`test:`/`refactor:`)
- [ ] Tests added or updated
- [ ] Documentation updated
- [ ] Changelog updated and version incremented (label: upcoming release)
- [ ] Github Issue linked if any
- [ ] Jira task updated

### CODE REVIEW

File: ionoscloud/resource_server_cube.go [hunk 1]
```diff
@@ -49,11 +49,6 @@ func resourceCubeServer() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
-			"cpu_family": {
```
Comment (cristiGuranIonos) [inline ionoscloud/resource_server_cube.go:52]: If this is in the docs, remove it

File: ionoscloud/resource_share.go [hunk 2]
```diff
@@ -53,9 +53,9 @@ func resourceShareCreate(ctx context.Context, d *schema.ResourceData, meta inter
 		Properties: &ionoscloud.GroupShareProperties{},
 	}
 
-	tempSharePrivilege := d.Get("edit_privilege").(bool)
+	tempSharePrivilege := d.Get("share_privilege").(bool)
 	request.Properties.SharePrivilege = &tempSharePrivilege
-	tempEditPrivilege := d.Get("share_privilege").(bool)
+	tempEditPrivilege := d.Get("edit_privilege").(bool)
```
Comment (cristiGuranIonos) [inline ionoscloud/resource_share.go:58]: add details to changelog

File: ionoscloud/resource_server_cube.go (hunk 1 above)
Comment (gabisavu-ionos) [reply ionoscloud/resource_server_cube.go:52]: removed

File: ionoscloud/resource_share.go (hunk 2 above)
Comment (gabisavu-ionos) [reply ionoscloud/resource_share.go:58]: added

**Review by cristiGuranIonos — APPROVED** [review]

**Pull request merged**